src/pages/about/about.js

```diff
@@ -13,6 +14,20 @@ export default function About() {
   const teamImages = [ PlaceHolder, PlaceHolder, PlaceHolder, PlaceHolder, PlaceHolder, 
                       PlaceHolder, PlaceHolder, PlaceHolder, PlaceHolder, PlaceHolder, PlaceHolder, PlaceHolder ]
 
+    const { formdata, handleChange } = useForm({
+    firstName: '',
+    lastName: '',
+    email: '',
+    phone: '',
+    message: '',
+  })
+
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    window.alert(`Submitting ${JSON.stringify(formdata, null, 2)}`)
+  }
+
```

Comment:
The Contact form should be a self standing component so that it can be reused in other places on the site. This bit of code should be refactored to that component.